crates/pathfinder/src/state/contract_hash.rs

```diff
@@ -284,7 +284,7 @@ impl serde_json::ser::Formatter for PythonDefaultFormatter {
     }
 }
 
-mod json {
+pub(crate) mod json {
```

Comment:
Why is this needed?

Reply:
It's used in [this test](https://github.com/eqlabs/pathfinder/pull/311/files#diff-9031c45e5d044575e5787fd8b394d0db51c03c689dcbb7333b25651f5d490af4R1100), to provide additional checks should the API change. Ofc makes sense only when run against a live api then.

Reply:
Perhaps the fn success() could just handle the sequencer api portion, and just return whatever simple short bytes, like "hello world"?

Reply:
Aah but the hello world idea wouldn't work against real sequencer. And the existing compressed fixture might not be live anywhere (it's the truncated `test.cairo` to keep things small). Phew. You could just ensure that's is serde_json::Value::Object perhaps?

Reply:
`{"hello":"world"}` for the mock payload and asserting for `serde_json::Value::Object` will work just fine in both cases :+1: .